### ISSUE TITLE
Loss function doesn't match paper

### ISSUE THREAD

**Issue by Krasner** [ajrcampbell/early-exit-ensembles#3]
Hi,

`ExitWeightedCrossEntropyLoss` doesn't match the loss function described in Equation 8. Specifically that equation doesn't have the term `gamma`:
`loss += self.alpha[ex] * gamma[ex] * F.cross_entropy(exit_logits, labels)`
In run.py the gamma argument isn't provided: `loss = criterion(logits, y)`  but the __call__ method for this function expects it:
```
def __call__(self, logits, labels, gamma):
```

Additionally, I don't see an implementation of diversity loss in equation 9.

Can you also upload the hydra config yaml file?

Thanks!

**Comment by ajrcampbell** [comment]
Hi @Krasner, 

The `gamma` term in the loss function is left over from experiments that were not included in final paper. In the yaml file, that I will commit, it is simply set to 1.  It should really be removed. Sorry for the confusion.

We removed the diversity loss as it negatively impacted the final results when `beta>0`. See the caption of Table 2. 

Thanks!
